api/src/main/java/org/eclipse/microprofile/lra/annotation/Status.java

```diff
@@ -35,40 +36,86 @@
  * or compensation notification but if the participant (the class that
  * contains the Compensate and Complete annotations) does not
  * support idempotency then it must be able to report its' status by
- * by annotating one of the methods with this <em>@Status</em>.
+ * by annotating one of the methods with this <code>&#64;Status</code>.
  * The annotated method should report the status according to one of the
  * {@link ParticipantStatus} enum values.
+ * </p>
  *
+ * <p>
  * If the annotated method is a JAX-RS resource method the id of the currently
  * running LRA can be obtained by inspecting the incoming JAX-RS headers. If
  * this LRA is nested then the parent LRA MUST be present in the header with the name
  * {@link org.eclipse.microprofile.lra.annotation.ws.rs.LRA#LRA_HTTP_PARENT_CONTEXT_HEADER}
  * and value is of type {@link java.net.URI}.
+ * </p>
  *
+ * <p>
  * If the annotated method is not a JAX-RS resource method the id of the currently
  * running LRA can be obtained by adhering to a predefined method signature as
  * defined in the LRA specification document. Similarly the method may determine
  * whether or not it runs with a nested LRA by providing a parameter to hold the parent id.
  * For example,
+ * </p>
+ *
  * <pre>
  *     <code>
  *          &#64;Status
  *          public void status(URI lraId, URI parentId) { ...}
  *     </code>
  * </pre>
+ *
+ * <p>
  * would be a valid status method declaration. If an invalid signature is detected 
  * the implementation of this specification MUST prohibit successful startup of the application
  * (e.g. with a runtime exception).
+ * </p>
  *
+ * <p>
  * If the participant has already responded successfully to an invocation
- * of the <em>@Compensate</em> or <em>@Complete</em> method then it may
- * report <em>404 Not Found</em> HTTP status code or in case of 
- * non-JAX-RS method returning {@link ParticipantStatus} to return <em>null</em>. 
+ * of the <code>&#64;Compensate</code> or <code>&#64;Complete</code> method then it may
+ * report <code>404 Not Found</code> HTTP status code or in case of
+ * non-JAX-RS method returning {@link ParticipantStatus} to return <code>null</code>.
  * This enables the participant to free up resources.
+ * </p>
  *
+ * <p>
  * Since the participant generally needs to know the id of the LRA in order
  * to report its status there is generally no benefit to combining this
- * annotation with the `@LRA` annotation (though it is not prohibited).
+ * annotation with the <code>&#64;LRA</code> annotation (though it is not prohibited).
+ * </p>
+ *
+ * <p>
+ * If the method is a JAX-RS resource method (or is a non JAX-RS method
+ * annotated with <code>&#64;Status</code> with return type
+ * <code>javax.ws.rs.core.Response)</code> then the following are the only
```

Comment:
parenthesis in <code> block

Reply:
Copy and paste propagates errors, shame we can't have modular javadoc